### ISSUE TITLE
[Top Bar] Issue Name

### ISSUE THREAD

**Issue by imaia** [zurb/foundation-sites#7069]
**How can we reproduce this bug?**

Create the top-menu example; hover mouse over dropdown-menu; dropdown-menu-items do not show;

1. Create html
2. Link assets
3. Copy example

**What did you expect to happen?**
Same behavior as with the example, where the dropdown menu works

**What happened instead?**
Nothing, not even an error

**Comment by aibrean** [comment]
Are you loading all the JS dependencies and initializing foundation?

**Comment by imaia** [comment]
I'm loading foundation.min.js provided by the complete download in http://foundation.zurb.com/sites/download.html/

**Comment by aibrean** [comment]
You're not getting a console error?

BTW you'll need to use markdown for code so I can see it.

**Comment by imaia** [comment]
Below should work, right?
```
<div class="top-bar">
<div class="row">
  <div class="top-bar-left">
    <ul class="dropdown menu" data-dropdown-menu>
      <li class="menu-text">Site Title</li>
      <li class="has-submenu">
        <a href="#">One</a>
        <ul class="submenu menu vertical" data-submenu>
          <li><a href="#">One</a></li>
          <li><a href="#">Two</a></li>
          <li><a href="#">Three</a></li>
        </ul>
      </li>
      <li><a href="#">Two</a></li>
      <li><a href="#">Three</a></li>
    </ul>
  </div>
  <div class="top-bar-right">
    <ul class="menu">
      <li><input type="search" placeholder="Search"></li>
      <li><button type="button" class="button">Search</button></li>
    </ul>
  </div>
</div>
</div>
```

**Comment by aibrean** [comment]
It does for me in my development environment. I do have jQuery and jQuery UI running separate from the Foundation provided libraries.

**Comment by zurbchris** [comment]
foundation.min.js from the customizer is not working currently. We fixed the issue for v6.0.1, so if you use the bower/npm package it should work just fine using the file from `/dist`

**Comment by zurbchris** [comment]
https://github.com/zurb/foundation-sites/issues/7088
